ui/guest/src/react/UltraStyledTooltip.tsx

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (C) 2007-2024 Crafter Software Corporation. All Rights Reserved.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as published by
+ * the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { Tooltip, TooltipProps } from '@mui/material';
+
+export const UltraStyledTooltip = ({ title, children, ...props }: TooltipProps) => {
+  return (
+    <Tooltip slotProps={{ tooltip: { sx: { fontSize: 12 } } }} {...props} title={title}>
+      {children}
+    </Tooltip>
+  );
+};
```

Comment:
Why not done through the styled API?

Reply:
For some reason the styles for the popper are not applying through sx styles. I tried on an example provided from mui.com and doesn't work. https://codesandbox.io/s/jovial-wescoff-23jhhj?file=/src/Demo.tsx
Tooltip classes doc: https://mui.com/material-ui/api/tooltip/#classes.

Reply:
Using slotProps to set the styles for the tooltip works